app/uk/gov/hmrc/tai/util/TaxAccountCalculator.scala

```diff
@@ -19,9 +19,12 @@ package uk.gov.hmrc.tai.util
 import uk.gov.hmrc.tai.model.domain.AllowanceComponentType
 import uk.gov.hmrc.tai.model.domain.calculation.CodingComponent
 
-trait TaxAccountCalculator {
+abstract class TaxAccountCalculator {
+  def taxFreeAmount(codingComponents: Seq[CodingComponent]): BigDecimal
```

Comment:
Just curious as to why you made this an abstract class. Also the same code exists in TAI under TaxAccountSummaryService. Could be a back end concern. Thoughts??

Reply:
Think we were just playing? @oc-ste